Lib/test/test_types.py

```diff
@@ -2009,6 +2010,24 @@ def test_equal(self):
         self.assertEqual(ns1, ns2)
         self.assertNotEqual(ns2, types.SimpleNamespace())
 
+    def test_richcompare(self):
```

Comment:
```suggestion
    def test_richcompare_unsupported(self):
```

That would solve the issue with the lack of == and != being tested.